### DISCUSSION TITLE
MacBook Pro 16 power draw

### DISCUSSION THREAD

**Discussion by byrnes** [cyclingwithelephants/devs-on-a-boat#17] [General]
In my usage with my MBP (Intel i9, 16") I see a sustained load of 100w under load, and the battery even discharges at 100% load. While idle, it's around 15-20w. Also, while charging I see a power draw of ~80-100w. This may change some of your assumptions about laptop power, you may need more power if your machine is under load for prolonged periods of time.

**Comment by cyclingwithelephants** [comment]
Thanks so much for this info. I'll definitely be factoring this in, I intend to overhaul the power section today

**Comment by bergie** [comment]
I measured the power consumption of a MacBook Pro 13” for a regular 8h work day yesterday: 0.1kWh, making it average use of 12.5W (or just over 1A @ 12VDC)

**Reply by cyclingwithelephants** [reply]
Awesome, thanks for this. Can you please confirm:
  - the model year
  - whether or not you're compiling code all day (or are you e.g. DevOps and therefore using the laptop more akin to a thin client)

I've got a 2020 intel MBP 13" so it would be great to know whether or not I need to measure my usage!

**Reply by bergie** [reply]
2019 MBP13" running Big Sur. This was a pretty heavy Zoom meeting day, not much compiling but most of the day on a video call

**Comment by cyclingwithelephants** [comment]
~~Have you tried things mentioned [here](https://www.gearslutz.com/board/music-computers/1309863-macbook-pro-16-i9-battery-drain-while-plugged.html)? I just want to make 100% this is an apple issue rather than for example the cable being plugged into the  "wrong" USBC port~~

I've done more research and this is very much a thing with this specific model
